src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,68 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2022-03-23"
+weight: 26
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`. For information about known limitations, see [Known Limitations](#known-limitations). For more information about CIS security compliance for Kubernetes, see the [CIS benchmark information](https://www.cisecurity.org/benchmark/kubernetes).
+
+The following settings are changed when `cisCompliance` is set to `true`:
+
+**Primary node configuration:**
+
+The admin.conf file ownership is set to `root:root`.
+
+**API server configuration:**
+
+* `--kubelet-certificate-authority` is set as appropriate.
+* The admission control plugin `PodSecurityPolicy` is enabled.
+* `--insecure-port` is set to `0`.
+
+**Kubelet configuration:**
+
+`--protect-kernel-defaults` is set to `true`.
+
+## Example YAML
+
+This YAML file example shows a valid specification for CIS compliance:
+
+```
+apiVersion: "cluster.kurl.sh/v1beta1"
+kind: "Installer"
+metadata:
+ name: "latest"
+spec:
+ kurl:
+   cisCompliance: true
```

Comment:
It feels like this field should be under `kubernetes` not `kurl`? Or are we ensuring that all add-ons are CIS compliant also?

Reply:
that is a very good suggestion, and I would be open to changing it
and we should change it before releasing the docs, if we're going to do so

Reply:
Currently it is under `kurl` This fixes all the CIS failures on some of the K8s components -  `apiserver, controller manager, kubelet`  basically all the failures spit out by the CIS benchmark. We can change the code to make the flag to go under `kubernetes` if that makes more sense.  On the flip side this is something `kurl` is fixing for kubernetes to meet the Security compliance.. for ex., we set kernel flags to prevent kubelet crashing from kernel panics. so it feels good to be under `kurl`.  But marc's suggestion feels right. 

```

+ ./kube-bench --config-dir=//cfg --config=//cfg/config.yaml --exit-code=1
[INFO] 1 Master Node Security Configuration
[INFO] 1.1 Master Node Configuration Files
[PASS] 1.1.1 Ensure that the API server pod specification file permissions are set to 644 or more restrictive (Automated)
[PASS] 1.1.2 Ensure that the API server pod specification file ownership is set to root:root (Automated)
[PASS] 1.1.3 Ensure that the controller manager pod specification file permissions are set to 644 or more restrictive (Automated)
[PASS] 1.1.4 Ensure that the controller manager pod specification file ownership is set to root:root (Automated)
[PASS] 1.1.5 Ensure that the scheduler pod specification file permissions are set to 644 or more restrictive (Automated)
[PASS] 1.1.6 Ensure that the scheduler pod specification file ownership is set to root:root (Automated)
[PASS] 1.1.7 Ensure that the etcd pod specification file permissions are set to 644 or more restrictive (Automated)
[PASS] 1.1.8 Ensure that the etcd pod specification file ownership is set to root:root (Automated)
```

Reply:
@marccampbell this is fixed in https://github.com/replicatedhq/kURL/pull/2783/